Dockerfile-proxy

```diff
@@ -1,15 +1,21 @@
-ARG RUNTIME_IMAGE=debian:stretch-20190812-slim
+ARG ARCH=amd64
+ARG RUNTIME_IMAGE=$ARCH/debian:buster-20200422-slim
```

Comment:
Perhaps update to `buster-20200514-slim` which is what is now used for buster in other places in the codebase.